_posts/en/newsletters/2020-03-04-newsletter.md

```diff
@@ -0,0 +1,181 @@
+  related [rust-secp256k1 issue][rust-secp256k1 #196].
+
+- [BIPs #886][] updates [BIP340][] schnorr signatures with two
+  recommendations: (1) to include entropy in the nonce calculation
+  whenever a random number generator is available, and (2) to verify any
+  signatures produced by the software are valid before distributing them
+  to outside programs, at least whenever the extra computation and delay
+  isn't burdensome.  These two steps help ensure an attacker can't
+  recover a user's private key by obtaining invalid signatures with
+  reused nonces; see [Newsletter #83][news83 safety] for attack details.
+  For other proposed changes to BIP340, see the [news item][bip340
+  update] above in this week's newsletter.
+
+{% comment %}<!-- BOLTs #714 merged but reverted -->{% endcomment %}
+
+{% include references.md %}
+{% include linkers/issues.md issues="17985,17264,3490,1307,710,886" %}
+[bitcoin core 0.19.1]: https://bitcoincore.org/bin/bitcoin-core-0.19.1/
+[lnd 0.9.1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.9.1-beta.rc1
+[fournier poster]: https://github.com/LLFourn/taproot-ggm/blob/master/main.pdf
+[poelstra proof]: https://github.com/apoelstra/taproot/blob/master/main.pdf
+[random oracle]: https://en.wikipedia.org/wiki/Random_oracle
+[differential power analysis]: https://en.wikipedia.org/wiki/Power_analysis#Differential_power_analysis
+[bip-wtxid-relay]: https://github.com/sdaftuar/bips/blob/2020-02-wtxid-relay/bip-wtxid-relay.mediawiki
+[news83 safety]: /en/newsletters/2020/02/05/#safety-concerns-related-to-precomputed-public-keys-used-with-schnorr-signatures
+[news83 tiebreaker]: /en/newsletters/2020/02/05/#alternative-x-only-pubkey-tiebreaker
+[wuille update]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-February/017639.html
+[daftuar wtxid]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-February/017648.html
+[snigirev nonce]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-February/017655.html
+[sign to contract]: https://www.wpsoftware.net/andrew/secrets/slides.pdf
+[secp s2c]: https://github.com/bitcoin-core/secp256k1/pull/589
+[secp nonce]: https://github.com/bitcoin-core/secp256k1/pull/590
+[ecdh]: https://en.wikipedia.org/wiki/Elliptic_curve_Diffie-Hellman
+[ecies]: https://en.wikipedia.org/wiki/Integrated_Encryption_Scheme
+[bip340 update]: #updates-to-bip340-schnorr-keys-and-signatures
+[news83 eclair determ]: /en/newsletters/2020/02/05/#eclair-1295
+[secp256k1_ecdh]: https://github.com/bitcoin-core/secp256k1/blob/96d8ccbd16090551aa003bfa4acd108b0496cb89/src/modules/ecdh/main_impl.h#L29-L69
+[rust-secp256k1 ecdh ret]: https://github.com/rust-bitcoin/rust-secp256k1/blob/master/src/ecdh.rs#L162
+[ruffing concern]: https://gist.githubusercontent.com/harding/603c2b18241bf61bb0bbe7a0383cf1c9/raw/20656e901472f217d1faa381ddda1d11214900da/foo.txt
+[710 comment]: https://github.com/bitcoin-core/secp256k1/pull/710/files#r370987476
```

Comment:
This link doesn’t appear to work anymore. Perhaps the comment was resolved.

Reply:
Pushed updated link that works for me.